### PR TITLE
Bugfix/combobox list items with multiple spaces get invalid

### PR DESCRIPTION
**Pull request by it-vegard**
### Description

Combobox: Fix issue where list items for options with multiple spaces get invalid IDs (including whitespace)

### CODE REVIEW

File: @navikt/core/react/src/form/combobox/FilteredOptions/filtered-options-util.ts [hunk 1]
```diff
@@ -19,9 +19,7 @@ const getFilteredOptionsId = (comboboxId: string) =>
   `${comboboxId}-filtered-options`;
 
 const getOptionId = (comboboxId: string, option: string) =>
-  `${comboboxId.toLocaleLowerCase()}-option-${option
-    .replace(" ", "-")
-    .toLocaleLowerCase()}`;
+  `${comboboxId}-option-${option}`.replace(/\s/g, "-").toLocaleLowerCase();
```
Comment (it-vegard) [inline @navikt/core/react/src/form/combobox/FilteredOptions/filtered-options-util.ts:22]: Did a minimal fix here, but could rewrite more of these functions to also take account for comboboxId potentially including space?

File: @navikt/core/react/src/form/combobox/FilteredOptions/filtered-options-util.test.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { describe } from "vitest";
+import filteredOptionsUtil from "./filtered-options-util";
+
+describe("filtered options util", () => {
```
Comment (it-vegard) [inline @navikt/core/react/src/form/combobox/FilteredOptions/filtered-options-util.test.ts:4]: Only added a test for the code I changed. Could add tests for the rest of the file as well?

Comment (HalvorHaugan) [reply @navikt/core/react/src/form/combobox/FilteredOptions/filtered-options-util.test.ts:4]: I don't think we need tests for such simple functions.

File: @navikt/core/react/src/form/combobox/FilteredOptions/filtered-options-util.ts (hunk 1 above)
Comment (HalvorHaugan) [reply @navikt/core/react/src/form/combobox/FilteredOptions/filtered-options-util.ts:22]: I don't think we need to do that.

**Review by HalvorHaugan — APPROVED** [review]

**Pull request merged**